src/client/object/ObjectFeatures/ObjectFeaturesItem.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+const ObjectFeaturesItem = ({ feature, wobjPermlink }) => {
+  const [showMore, setShowMore] = useState(false);
+  const toggleShowMore = () => setShowMore(!showMore);
+  const stringLengthLimit = 60;
+
+  useEffect(() => {
+    if (feature.value.length > stringLengthLimit) setShowMore(true);
```

Comment:
feature.value.length > stringLengthLimit set in variable, because you used it on 20th string